src/components/avatar/index.ts

```diff
@@ -0,0 +1 @@
+export { Avatar } from './avatar';
```

Comment:
I think that the avatar is not exported in the parent index.ts of src/components. Please add this so the avatar component is also included in the NPM package when being built.

Reply:
@malinovic fixed